gef-ui/server/routes/bank-review-date.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import express from 'express';
+import { getBankReviewDate, postBankReviewDate } from '../controllers/bank-review-date';
+import { validateRole, validateToken, validateBank } from '../middleware';
+import { MAKER } from '../constants/roles';
+
+const router = express.Router();
```

Comment:
should this file be called `application-bank-review-date.ts`? Would that make it easier to see in the routes folder?

Reply:
The application is the deal, but this is about the facility. Do agree the routes folder is a nightmare though

Reply:
Would it make sense to move all of the pages in this journey into a `facility` folder?

Reply:
maybe `facility-bank-review-date` if you want a cheeky easy win, but a folder would be better